src/docs.js

```diff
@@ -1,4 +1,35 @@
-function documenter() {
+let tar = require('tar-stream');
+
+async function documenter(options = {}) {
+  // print schemas
+  console.log(options.schemas);
+
+  // take schemas(name + content) + add it to stream tarball
+  // pack is a streams2 stream
+  let tarball = tar.pack();
+
+  let schemas = options.schemas;
+
+  if (schemas) {
+    for (let schema of schemas) {
+      tarball.entry({name: schema.name}, schema.content);
+    }
+  }
+  tarball.finalize();
+
+  let output = {
+    tarball,
+  };
+
+  return output;
+
+  // add docs to tarball
```

Comment:
jonasfj : Ok cool. I thought it was going to be an issue of consistency that I am using schema.id and docs filenames inside the tarball